docs/releases/network-upgrades.md

```diff
@@ -25,13 +25,14 @@ To help ensure Quality of Service on the network by providing the ability to gen
-| TBD  | Node release | Nano node VXX.X released to include new PoW algorithm in PoW server and final changes allowing transition via epoch blocks. |
-| TBD | <span class="no-break">Epoch blocks start</span> | Distribution of epoch blocks to each account which upgrades them to use the new PoW algorithm.  Once an account is upgraded nodes will only validate work made using the new PoW algorithm for that account. |
-| TBD | Epoch blocks end | Distribution of epoch blocks ends after all accounts are upgraded. |
+| <span class="no-break">2023-MM-DD</span> | Node release | The PoW server was discontinued in V25.0, as there are better implementations made by the community. Starting on node's version V25.0, its release does not bundle it anymore.                                                                                                                                                                                                                                                                                                     |
+| TBD                                      | Node release | Nano node VXX.X released to include new PoW algorithm in PoW server and final changes allowing transition via epoch blocks.                                                                                                                                                                                                                                                                                                                                                        |
+| TBD                                      | <span class="no-break">Epoch blocks start</span> | Distribution of epoch blocks to each account which upgrades them to use the new PoW algorithm.  Once an account is upgraded nodes will only validate work made using the new PoW algorithm for that account.                                                                                                                                                                                                                                                                       |
+| TBD                                      | Epoch blocks end | Distribution of epoch blocks ends after all accounts are upgraded.                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
Is a new PoW algorithm still planned ?
Shouldn't we rather talk about the bounded backlog and the featuresrequired to reach "commercial grade"

Reply:
Yes, just realized that the changes proposed here are incorrect. What happened is that we only stopped bundling the PoW server in the Docker container, that was the old implementation written in C++.